tests/jobs/test_scheduler_job.py

```diff
@@ -4542,8 +4543,7 @@ def test_do_schedule_max_active_runs_and_manual_trigger(self, dag_maker, mock_ex
         session = settings.Session()
         dag_version = DagVersion.get_latest_version(dag.dag_id)
         dag_run = dag_maker.create_dagrun(state=State.QUEUED, session=session, dag_version=dag_version)
-
-        DAG.bulk_write_to_db("testing", None, [dag], session=session)  # Update the date fields
```

Comment:
Do we still need this?

Reply:
I decided to leave it so the dag does not exist without being associated with a bundle